pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```diff
@@ -186,6 +189,10 @@ func NewInformers(versionedInformers clientgoinformers.SharedInformerFactory, lo
 	if err != nil {
 		return nil, err
 	}
+	authzClient, err := authzclient.NewForConfig(jsonLoopbackClientConfig)
```

Comment:
this does make it so that all future authz informers will end up using JSON.  just something to think about

Reply:
there's a comment above explaining that CRDs require JSON